Makefile

```diff
@@ -24,6 +27,14 @@ install_tools:
 lint:
 	scripts/lint.sh
 
+test: unit_test integration_test
+
+unit_test:
```

Comment:
Unit test is failing in CI, could you take a look?

Reply:
Yeah, sure.